src/vorta/models.py

```diff
@@ -106,6 +106,9 @@ class Meta:
 class SourceFileModel(pw.Model):
     """A folder to be backed up, related to a Backup Configuration."""
     dir = pw.CharField()
+    dir_size = pw.IntegerField()
+    dir_files_count = pw.IntegerField()
+    dir_type_isdir = pw.BooleanField()
```

Comment:
This variable name isn't good, might change it...

Reply:
Change to `path_isdir`?

Reply:
changed, first wanted to stick to the "dir_" prefix, but nvm